cSpell.json

```diff
@@ -5,18 +5,33 @@
         "browsable",
         "comparands",
         "Doomen",
+        "Nyrup",
```

Comment:
Do we really have to do this? Isn't there a better way to exclude URLs and such?

Reply:
Yes. I fiddled around a bit and came up with a new ignore pattern: `Contributors`